src/app/components/Button/button.scss

```diff
@@ -0,0 +1,44 @@
+@import '@/styles/index.scss';
+
+.button {
+  background-color: $color-primary;
+  color: #fff;
+  border-radius: 12px;
+  border: none;
+  width: 262px;
+  height: 56px;
+  padding: 10px;
+  text-align: center;
+  font-family: 'Poppins', sans-serif;
+  font-size: 20px;
+  font-weight: 500;
+  cursor: pointer;
+  transition: background-color 0.5s ease;
+
+  &--secondary {
+    width: 100%;
+  }
+
+  &--deactivated {
+    background-color: $color-button-disabled;
+    color: #aaa;
+    cursor: not-allowed;
+    pointer-events: none;
+  }
+
+  &:hover {
+    background-color: $color-button-hover;
+  }
+
+  &:focus {
```

Comment:
focus should have the same color like hover , u should add next class :active with background-color: $color-button-active;

Reply:
jednak sprawdzilem ten focus i po kliknieciu focus zostaje na elemencie wiec nie moze go byc wogole.